module/checker.go

```diff
@@ -420,6 +422,24 @@ func (m Module) checkMinMax(min, max *uint64) {
 		"`min` value is greater than `max` value")
 }
 
+func (m Module) checkLen(len, min, max *uint64) {
```

Comment:
I'm curious why `min_len` and `max_len` are required to be equal to `len` if present. Why not just forbid min and max from being set if `len` is provided?

Reply:
In previous version, set `min_len` and `max_len` to equal value means value should equal to, that's little complex and not clear.
So I suggest to add a new property `len`. 
In order to compatible the old usage, I kept `min_len` and `max_len`, while generating go/cpp code, it will only print equal condition.

Reply:
Understanding, I'll fix it